server/container_create.go

```diff
@@ -384,6 +384,27 @@ func ensureSaneLogPath(logPath string) error {
 	return nil
 }
 
+// addSecretsBindMounts mounts user defined secrets to the container
+func addSecretsBindMounts(mountLabel, ctrRunDir, secretsFilePath string, specgen generate.Generator) error {
+	// secretsFilePath is used to override the mount file path for testing purposes only
```

Comment:
? isn't this coming from the config or cmd line flag?!

Reply:
yes, it is coming from config. Will reword the comment.

Reply:
fixed